configure.ac

```diff
@@ -94,16 +94,10 @@ CHECK_VAR_REDZONE()
 CHECK_PKG_MPI([found_mpi="yes"], [found_mpi="no"])
 
 AC_ARG_ENABLE([tests],
-   [AS_HELP_STRING([--disable-tests], [Disable build of functional test binaries])])
-AS_IF([test "${enable_tests}" != "no" -a "${enable_neuron}" = "yes"],
-      [AC_MSG_WARN([Disabling tests due to Neuron configuration.])
-       enable_tests=no],
-      [test "${enable_tests}" = "yes" -a "${found_mpi}" = "no"],
-      [AC_MSG_ERROR([Tests requested, but MPI not found.  Aborting.])],
-      [test "${found_mpi}" = "no"],
-      [AC_MSG_WARN([Disabling tests due to no MPI.])
-       enable_tests=no])
-AM_CONDITIONAL([ENABLE_TESTS], [test "$enable_tests" != "no"])
+   [AS_HELP_STRING([--enable-tests], [Enable build of functional test binaries (default: disabled)])])
```

Comment:
I would have done it the other way around, that is, check for `cuMemHostAlloc`, and if not found, disable tests. That way, we don't break any workflows that use the tests. (I think DLAMI runs the tests as part of release prep.)

Reply:
I'd rather kill dependency on the tests, which are really lame and pointless to begin with.  I think a side-effecty test build sounds more likely to confuse customers.